### ISSUE TITLE
Unable to create temporary channel from ICE interface

### ISSUE THREAD

**Issue by daveoon** [mumble-voip/mumble#3516]
Can anyone advise how to create a temporary channel from ICE interface? I m using the methods describe below but it doesn't create a temporary channel but a permanent one. Is it only working on the client side mumble.protocol only?
```
const channel_id = await server.addChannel('temp', 0)
const channel = await server.getChannelState(channel_id)
channel.name = 'Temp'
channel.temporary = true
await server.setChannelState(channel)
```

**Comment by tu-maurice** [comment]
Wouldn't a temporary channel be removed instantly as there are no users in it?
Could you explain further what you're trying to achieve?

**Comment by daveoon** [comment]
@tu-maurice Hi, i understand from the client side that when they create a temp channel, they will be move into the channel automatically. I want to achieve the same thing from the mumble protocol without user intervention. Is this possible from the ICE interface or the channel temporary state is only a read status?

**Comment by tu-maurice** [comment]
Well I'm not overly confident in my knowledge of the source, but from what I can see the call to setChannelState does end up at [impl_Server_setChannelState](https://github.com/mumble-voip/mumble/blob/b44b1f2172a8fa489697cd78d2cc92254c968283/src/murmur/MurmurIce.cpp#L1254) which then calls [Server::setChannelState](https://github.com/mumble-voip/mumble/blob/6195761d290fd3ba4296215e525da8008ddf86d2/src/murmur/RPC.cpp#L215).

And neither of them seems to care about the bTemporary member variable of the Channel class, so....

TL;DR: It seems to be read only.

**Comment by daveoon** [comment]
@tu-maurice thanks for the confirmation. seems like i need to look for workaround to achieve what i needed. Perhaps using a bot on the client side to achieve this should be possible. appreciate for taking the time to answer my question. Cheers!
